docs/Installation-Windows.md

```diff
@@ -0,0 +1,190 @@
+
+## Step 1: Install Python via Anaconda
+
+<a href="https://www.anaconda.com/download/#windows" target="_blank">Download</a> and install Anaconda for Windows.  By using Anaconda, you can manage separate environments for different distributions of Python.  Python 3 is required as we no longer support Python 2.  In this guide, we are using Python version 3.6 and Anaconda version 5.1 ([64-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86_64.exe) or [32-bit](https://repo.continuum.io/archive/Anaconda3-5.1.0-Windows-x86.exe) direct links).
+
+<p align="center">
+    <img src="images/anaconda_install.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+We recommend the default _advanced installation options_. However, select the options appropriate for your specific situation.
+
+<p align="center">
+    <img src="images/anaconda_default.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+After installation, you must open __Anaconda Navigator__ to finish the setup. From the Windows search bar, type _anaconda navigator_. You can close Anaconda Navigator after it opens.
+
+## Step 2: Setup and Activate a New Conda Environment
+
+You will create a new [Conda environment](https://conda.io/docs/) to be used with ML-Agents.  This means that all the packages that you install are localized to just this environment.  It will not affect any other installation of Python or other environments.  Whenever you want to run ML-Agents, you will need activate this Conda environment.
+
+To create a new Conda environment, open a new Anaconda Prompt (_Anaconda Prompt_ in the search bar) and type in the following command:
+
+```
+conda create -n ml-agents python=3.6
+```
+
+You may be asked to install new packages. Type `y` and press enter _(make sure you are connected to the internet)_. You must install these required packages. The new Conda environment is called ml-agents and uses Python version 3.6.
+
+<p align="center">
+    <img src="images/conda_new.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+To use this environment, you must activate it. _(To use this environment In the future, you can run the same command)_. In the same Anaconda Prompt, type in the following command:
+
+```
+conda activate ml-agents
+```
+
+You should see `(ml-agents)` prepended on the last line.
+
+Next, install `tensorflow` and `tensorflow-gpu`. Install these packages using `pip` - which is a package management system used to install Python packages. In the same Anaconda Prompt, type in the following command _(make sure you are connected to the internet)_:
+
+```
+pip install tensorflow tensorflow-gpu
```

Comment:
If using only CPU, then they should use `tensorflow` if installing cuda, etc, then use `tensorflow-gpu`.

Reply:
done